src/components/SettingsDialog/ExportBoard/ExportHintHiddenColumns/ExportHintHiddenColumns.scss

```diff
@@ -0,0 +1,48 @@
+@import "src/constants/style.scss";
+
+$icon-text--gap: 20px;
+$columns-list--padding-left: $icon-text--gap + 50px;
+$list-style-type: "– ";
+
+.export-board__hint-hidden-columns {
+  margin-top: $margin--large;
+  padding: 0 $padding--large 0 $padding--large;
+
+  font-size: $text-size--medium;
+}
```

Comment:
Personally I'd move this to `ExportBoard.scss`. This defines the hint component, but is used in the `ExportBoard.tsx` component. So I think it is in better hands there.

Reply:
yea true, I now just removed the class because it didn't really have a job anyways.